app/controllers/items_controller.rb

```diff
@@ -2,16 +2,16 @@ class ItemsController < ApplicationController
     def index
         @items=Item.all
```

Comment:
add spacing around the `=` like `@items = Item.all`. Same for all the lines below.

Reply:
[Style]
If you're curious, [here's the official style guide note on adding spacing around operators](https://github.com/rubocop-hq/ruby-style-guide#spaces-operators).

Please also be sure to use an [indentation of 2 spaces, not 4](https://github.com/rubocop-hq/ruby-style-guide#spaces-indentation).